crypto/crypto_aes_aesni_m128i.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _CRYPTO_AES_AESNI_M128I_H_
+#define _CRYPTO_AES_AESNI_M128I_H_
+
+#include <emmintrin.h>
+
+/**
+ * crypto_aes_encrypt_block_aesni_m128i(in, key):
+ * Using the expanded AES key ${key}, encrypt the block ${in} and return the
+ * resulting ciphertext.  This implementation uses x86 AESNI instructions,
+ * and should only be used if CPUSUPPORT_X86_AESNI is defined and
+ * cpusupport_x86_aesni() returns nonzero.
+ */
+__m128i crypto_aes_encrypt_block_aesni_m128i(const __m128i, const void *);
```

Comment:
Is the `const` in `const __m128i` meaningful?

Reply:
Not to my knowledge.  It was partly because (in my mind) that reinforced that it was an input parameter, and partly just because `void crypto_aes_encrypt_block_aesni(const uint8_t[16], uint8_t[16], const void *);` had one there.  I'll remove it.